hamcrest/Hamcrest/Matchers.php

```diff
@@ -8,6 +8,15 @@
 
```

Comment:
This is a generated file.

Were changes in this file a result of running the dedicated build script?